src/containers/GeneralInfo/GeneralInfo.tsx

```diff
@@ -55,7 +63,7 @@ export const GeneralInfo = ({
                 <div className="general-info-stats lower">
                     <div className="stat">
                         <label>node version</label>
-                        <p>{nodeInfo && nodeInfo.version}</p>
+                        <NodeVersion version={nodeInfo && nodeInfo.version} updateAvailabe={compareNodeVersions()} />
```

Comment:
You can pass to this component currentVersion and latestVersion and then do the comparing there